index.js

```diff
@@ -28,10 +28,12 @@ const getAsString = (string, options) => string.replace(groupedIssueRegex, match
 
 const getAsDocumentFragment = (string, options) => {
 	const fragment = document.createDocumentFragment();
-	for (const [index, text] of Object.entries(string.split(groupedIssueRegex))) {
-		if (index % 2) { // URLs are always in odd positions
+	const parts = string.split(groupedIssueRegex);
+
+	for (const [index, text] of parts.entries()) {
+		if (index % 5 === 1) { // Issues are always first in capturing group
 			fragment.append(domify(linkify(text, options)));
-		} else if (text.length > 0) {
+		} else if (index % 5 === 0 && text.length > 0) { // Text is always in split position
```

Comment:
Neither comment here explains the situation. It should be more like:

> Every regex group appears in the split strings
> At position 5n+1 there's the full matched issue string
> At position 5n+0 there's what doesn't match the regex

Reply:
[`8325460` (#13)](https://github.com/sindresorhus/linkify-issues/pull/13/commits/8325460e646bc3b94cc093e955dd01c8a4552fd8)